pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -148,12 +151,20 @@ func (r *ReplicaSetReconciler) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	r.log.Debugf("Ensuring StatefulSet is ready, with type: %s", getUpdateStrategyType(mdb))
-	if ready, err := r.isStatefulSetReady(mdb, getUpdateStrategyType(mdb)); err != nil {
+	currentSts := appsv1.StatefulSet{}
+	if err := r.client.Get(context.TODO(), mdb.NamespacedName(), &currentSts); err != nil {
+		r.log.Warnf("Error getting StatefulSet: %s", err)
```

Comment:
[nit] do we want to use `r.log.WarnF` here? (Almost) everywhere else we use `r.log.InfoF`(except one case in which we use `ErrorF`) Maybe we should use the same everywhere for consistency?

(btw I would be more in favour of using `WarnF` instead of `InfoF`

Reply:
Yep we need to make sure we're consistent with our logging levels. 👍